middleware/dnssec/README.md

```diff
@@ -43,5 +43,7 @@ If monitoring is enabled (via the *prometheus* directive) then the following met
 
 * coredns_dnssec_cache_size{type} - total elements in the cache, type is "signature".
```

Comment:
then we should remove the `type` here as well. I added it to be in sync with caching, but you have a point in removing the type from *dnssec*

Sorry for nitpicking.